vmdb/app/models/miq_report/generator/sorting.rb

```diff
@@ -49,7 +49,6 @@ def build_sort_suffix_data
     rpt_sortby = self.sortby.to_miq_a
     rpt_sortby.each do |sb|
       col, sfx = sb.split("__")
-      dt = self.class.get_col_type(self.col_to_expression_col(sb))
```

Comment:
Ok, so I search for similar looking code, because I figured hey, copy-paste...found this:

https://github.com/ManageIQ/manageiq/blob/d8a443512d155b3f1dc044f40ca2591118310543/vmdb/app/models/miq_report/formatting.rb#L69-73

which leads to

https://github.com/ManageIQ/manageiq/blob/d8a443512d155b3f1dc044f40ca2591118310543/vmdb/app/models/miq_report/formatting.rb#L32

of which the last part of the last line says

```ruby
FORMAT_DEFAULTS_AND_OVERRIDES[:formats_by_data_type][dt]
```

So, I'm guessing dt means data_type...of the column.

Anyway, the important part I see is that the first link above is the format method, which is called below with `sb`.  In the first link, format derives its own `dt` from the `sb`, so I think we're `a-ok` here. Q.E.D.